sb3_contrib/common/maskable/buffers.py

```diff
@@ -0,0 +1,87 @@
+from typing import Generator, NamedTuple, Optional
+
+import numpy as np
+import torch as th
+from gym import spaces
+from stable_baselines3.common.buffers import RolloutBuffer
+from stable_baselines3.common.type_aliases import RolloutBufferSamples
+from stable_baselines3.common.vec_env import VecNormalize
+
+# NamedTuple doesn't support straightforward subclassing, but it can be
```

Comment:
why not re-defining `MaskableRolloutBufferSamples` completely, without deriving from `RolloutBufferSamples` ?